src/views/notifications/CancelAllNotification.tsx

```diff
@@ -0,0 +1,90 @@
+import { shallowEqual } from 'react-redux';
+import styled from 'styled-components';
+
+import { STRING_KEYS } from '@/constants/localization';
+import { CANCEL_ALL_ORDERS_KEY, LocalCancelAllData } from '@/constants/trade';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { AssetIcon } from '@/components/AssetIcon';
+import { Details } from '@/components/Details';
+import { LoadingSpinner } from '@/components/Loading/LoadingSpinner';
+// eslint-disable-next-line import/no-cycle
+import { Notification, NotificationProps } from '@/components/Notification';
+
+import { useAppSelector } from '@/state/appTypes';
+import { getMarketData } from '@/state/perpetualsSelectors';
+
+type ElementProps = {
+  localCancelAll: LocalCancelAllData;
+};
+
+export const CancelAllNotification = ({
+  isToast,
+  localCancelAll,
+  notification,
+}: NotificationProps & ElementProps) => {
+  const stringGetter = useStringGetter();
+  const isCancelForSingleMarket = localCancelAll.key !== CANCEL_ALL_ORDERS_KEY;
```

Comment:
nit,. as a follow up maybe we could add some extra code to detect if all the orders are attributed to a single market, even if the user isn't filtered to it